templates/contests_view.html.twig

```diff
@@ -51,7 +51,7 @@
 <div id="scores">
     <h3>{{ msg('scores') }}</h3>
     {% if can_view_scores %}
-        <table class="table">
+        <table class="table"  class="sortable-theme-bootstrap" data-sortable>
```

Comment:
There's already a `class` attribute; any new classes should be added to that.

Also, this table is the only one with the theme applied; is that intentional?